src/super_gradients/training/utils/tensor_formats/bbox_formats/cxcywh.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 import torch
 
-from super_gradients.training.utils.bbox_formats.bbox_format import (
+from super_gradients.training.utils.tensor_formats.bbox_formats.bbox_format import (
```

Comment:
What was the reasoning in moving 
`super_gradients.training.utils.bbox_formats.bbox_format` to from `super_gradients.training.utils.tensor_formats.bbox_formats` ?

Reply:
I though that it made sense since they are all related to formats, and grouping it together would increase module cohesion. But to be honest I am not fully sure that moving bbox_formats in this module is good, because its also a standalone module for bbox format conversion.
So, should we just take bbox_formats out  of tensor_formats like it was?

Reply:
In my opinion - yes. It is just unexpected to find `bbox_formats` under `tensor_formats`. 
Maybe rename tensor_formats to data_formats. Then import path would be more clear: utils -> data_formats -> bbox_formats.
Side note: I'd actually recommend to move it to `datasets` from `utils`:
`super_gradients.datasets.data_formats.bbox_formats` ❤️ ❤️ ❤️

Reply:
@Louis-Dupont overall, LGTM. Feel free to merge without this change if not feeling adventurous enough to include proposed change to this PR :)